content/hi/docs/tutorials/kubernetes-basics/_index.html

```diff
@@ -0,0 +1,117 @@
+            <div class="col-md-4">
+              <div class="thumbnail">
+                <a href="/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_01.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/"><h5>1. कुबेरनेट्स क्लस्टर बनाएं</h5></a>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-4">
+              <div class="thumbnail">
+                <a href="/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_02.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/"><h5>2. ऐप डिप्लॉय करें</h5></a>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-4">
+              <div class="thumbnail">
+                <a href="/docs/tutorials/kubernetes-basics/explore/explore-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_03.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/docs/tutorials/kubernetes-basics/explore/explore-intro/"><h5>3. अपना ऐप एक्सप्लोर करें</h5></a>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="col-md-12">
+          <div class="row">
+            <div class="col-md-4">
+              <div class="thumbnail">
+                <a href="/docs/tutorials/kubernetes-basics/expose/expose-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_04.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/docs/tutorials/kubernetes-basics/expose/expose-intro/"><h5>4. अपने ऐप को सार्वजनिक रूप से एक्सपोज़ करें</h5></a>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-4">
+              <div class="thumbnail">
+                <a href="/docs/tutorials/kubernetes-basics/scale/scale-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_05.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/docs/tutorials/kubernetes-basics/scale/scale-intro/"><h5>5. अपना ऐप बढ़ाएं</h5></a>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-4">
+              <div class="thumbnail">
+                <a href="/docs/tutorials/kubernetes-basics/update/update-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_06.svg?v=1469803628347" alt=""></a>
+                <div class="caption">
+                  <a href="/docs/tutorials/kubernetes-basics/update/update-intro/"><h5>6. अपना ऐप अपडेट करें</h5></a>
```

Comment:
```suggestion
                <a href="/hi/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_01.svg?v=1469803628347" alt=""></a>
                <div class="caption">
                  <a href="/hi/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/"><h5>1. कुबेरनेट्स क्लस्टर बनाएं</h5></a>
                </div>
              </div>
            </div>
            <div class="col-md-4">
              <div class="thumbnail">
                <a href="/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_02.svg?v=1469803628347" alt=""></a>
                <div class="caption">
                  <a href="/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/"><h5>2. ऐप डिप्लॉय करें</h5></a>
                </div>
              </div>
            </div>
            <div class="col-md-4">
              <div class="thumbnail">
                <a href="/hi/docs/tutorials/kubernetes-basics/explore/explore-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_03.svg?v=1469803628347" alt=""></a>
                <div class="caption">
                  <a href="/hi/docs/tutorials/kubernetes-basics/explore/explore-intro/"><h5>3. अपना ऐप एक्सप्लोर करें</h5></a>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div class="col-md-12">
          <div class="row">
            <div class="col-md-4">
              <div class="thumbnail">
                <a href="/hi/docs/tutorials/kubernetes-basics/expose/expose-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_04.svg?v=1469803628347" alt=""></a>
                <div class="caption">
                  <a href="/hi/docs/tutorials/kubernetes-basics/expose/expose-intro/"><h5>4. अपने ऐप को सार्वजनिक रूप से एक्सपोज़ करें</h5></a>
                </div>
              </div>
            </div>
            <div class="col-md-4">
              <div class="thumbnail">
                <a href="/hi/docs/tutorials/kubernetes-basics/scale/scale-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_05.svg?v=1469803628347" alt=""></a>
                <div class="caption">
                  <a href="/hi/docs/tutorials/kubernetes-basics/scale/scale-intro/"><h5>5. अपना ऐप स्केल करें</h5></a>
                </div>
              </div>
            </div>
            <div class="col-md-4">
              <div class="thumbnail">
                <a href="/hi/docs/tutorials/kubernetes-basics/update/update-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_06.svg?v=1469803628347" alt=""></a>
                <div class="caption">
                  <a href="/hi/docs/tutorials/kubernetes-basics/update/update-intro/"><h5>6. अपना ऐप अपडेट करें</h5></a>
```

Reply:
We can link to Hindi version as these pages are part of our initial localization